spec/fixtures/http.ts

```diff
@@ -39,15 +39,15 @@ export function mockRCVariableFetch(
   data: any,
   token: string = 'thetoken'
 ): nock.Scope {
-  let mock: nock.Scope = nock('https://runtimeconfig.googleapis.com').get(
+  let interceptor = nock('https://runtimeconfig.googleapis.com').get(
```

Comment:
Everything in this file seems to be a rename or a cleanup. Why is this necessary for `v3`?

Reply:
I had to do a small refactoring in this file so it compiles, since an updated version of `nock` ships with more strict (more correct) type definitions.

Renames were not strictly necessary. They follow names of interfaces defined by `nock` and I tried to make it more precise and descriptive. I can extract renames into a separate PR if you feel like it's too much changes to comfortably review and approve.